src/libs/mps_comm/Makefile

```diff
@@ -36,6 +38,13 @@ else
   WARN_TARGETS += warning_cpp17
 endif
 
+HAVE_MQTT = 1
```

Comment:
Mmh the packages dont have pkg-config files, so a proper check is not possible.
We plan on migrating the refbox to cmake soon, but until then we should proably at least to a small sanity check whether the mqtt libs exist. something like this:

HAVE_MQTT = $(shell if [ -f /usr/include/mqtt/async_client.h ]; then echo 1; else echo 0; fi)

Reply:
This change is also included in a cherry-pick so this thread is resolved.